docs(edge-services): reference Serverless Containers behind Edge Serv… (#6666)
scaleway/docs-content: pages/edge-services/quickstart.mdx

- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- Created either a [Load Balancer](/load-balancer/how-to/create-load-balancer/) or an [Object Storage bucket](/object-storage/how-to/create-a-bucket/)

<Message type="tip">

> You can also use a [Serverless Container or Serverless Function](/serverless-containers/how-to/deploy-container-behind-edge-services/) as an Edge Services backend. This feature is not available in the Scaleway console: see the guide to deploying a Serverless Container behind Edge Services for instructions using the CLI or Terraform.

Reviewer: ```suggestion
You can also use Serverless Containers and Serverless Functions as Edge Services backends. This feature is not available in the Scaleway console. See [How to deploy a Serverless Container behind Edge Services](/serverless-containers/how-to/deploy-container-behind-edge-services/) for instructions using the CLI or Terraform.
```
